### PR TITLE
[Feature] Light/Dark Submission Flow

### PR DESCRIPTION
**Pull request by eisvogel**
This PR updates the submission flow to the latest color scheme.

It includes:
* Color adjustments
* Minor layout fixes
* Update QR code scanner

### CODE REVIEW

**Review by johannesrohwer — APPROVED** [review]
I just clicked through the dark mode flow, it looks great!

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionQRScannerViewController.swift [hunk 1]
```diff
@@ -176,13 +192,59 @@ extension ExposureSubmissionQRScannerViewController: AVCaptureMetadataOutputObje
...
+	private func updatePreviewMaskIfNeeded() {
+		guard needsPreviewMaskUpdate else { return }
+		needsPreviewMaskUpdate = false
+
+		guard let previewLayer = previewLayer else { return }
+		guard focusView.backdropOpacity > 0 else {
+			previewLayer.mask = nil
+			return
+		}
+		let backdropColor = UIColor(white: 0, alpha: 1 - max(0, min(focusView.backdropOpacity, 1)))
```
Comment (johannesrohwer) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionQRScannerViewController.swift:212]: nice UI improvement

File: src/xcode/ENA/ENA/Source/Views/ExposureSubmission/ExposureSubmissionTestResultHeaderView.swift [hunk 2]
```diff
@@ -98,27 +91,31 @@ class ExposureSubmissionTestResultHeaderView: DynamicTableViewHeaderFooterView {
...
+		subTitleLabel.textColor = .enaColor(for: .textPrimary2)
 		subTitleLabel.translatesAutoresizingMaskIntoConstraints = false
 		column.addSubview(subTitleLabel)
 		subTitleLabel.leftAnchor.constraint(equalTo: column.leftAnchor, constant: 5).isActive = true
 		subTitleLabel.topAnchor.constraint(equalTo: barView.topAnchor).isActive = true
 
-		titleLabel = UILabel()
-		titleLabel.text = "title"
+		titleLabel = ENALabel()
+		titleLabel.text = "Title 2"
```
Comment (i336616) [inline src/xcode/ENA/ENA/Source/Views/ExposureSubmission/ExposureSubmissionTestResultHeaderView.swift:105]: Do we have/need an AppStrings entry for this?

**Review by i336616 — COMMENTED** [review]
There's a couple of strings being used for label.text values, do they need to be localized?

File: src/xcode/ENA/ENA/Source/Views/ExposureSubmission/ExposureSubmissionTestResultHeaderView.swift (hunk 2 above)
Comment (EisvogelSAP) [reply src/xcode/ENA/ENA/Source/Views/ExposureSubmission/ExposureSubmissionTestResultHeaderView.swift:105]: Not required, as they are just placeholders. Will be overridden when the header is configured anyway.

Comment (i336616) [reply src/xcode/ENA/ENA/Source/Views/ExposureSubmission/ExposureSubmissionTestResultHeaderView.swift:105]: 👍

**Review by i336616 — APPROVED** [review]
👍

**Pull request merged**